PolyEngine/Core/Src/Path.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "Defines.hpp"
+#include "Dynarray.hpp"
+#include "String.hpp"
+
+namespace Poly {
+
+	class CORE_DLLEXPORT Path
+	{
+	public:
+		static Path* Build();
```

Comment:
A common convention is for a `build()` method to return the final _built_ value, so this might be a little confusing.